To: vim_dev@googlegroups.com
Subject: Patch 7.3.411
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.411
Problem:    Pasting in Visual mode using the "" register does not work. (John
	    Beckett)
Solution:   Detect that the write is overwriting the pasted register.
	    (Christian Brabandt)
Files:	    src/normal.c


*** ../vim-7.3.410/src/normal.c	2011-10-04 21:22:40.000000000 +0200
--- src/normal.c	2012-01-23 22:13:44.000000000 +0100
***************
*** 9329,9335 ****
  # ifdef FEAT_CLIPBOARD
  	    adjust_clip_reg(&regname);
  # endif
! 	    if (regname == 0 || VIM_ISDIGIT(regname)
  # ifdef FEAT_CLIPBOARD
  		    || (clip_unnamed && (regname == '*' || regname == '+'))
  # endif
--- 9329,9335 ----
  # ifdef FEAT_CLIPBOARD
  	    adjust_clip_reg(&regname);
  # endif
! 	    if (regname == 0 || regname == '"' || VIM_ISDIGIT(regname)
  # ifdef FEAT_CLIPBOARD
  		    || (clip_unnamed && (regname == '*' || regname == '+'))
  # endif
*** ../vim-7.3.410/src/version.c	2012-01-23 20:48:34.000000000 +0100
--- src/version.c	2012-01-26 11:42:33.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     411,
  /**/

-- 
Engineers understand that their appearance only bothers other people and
therefore it is not worth optimizing.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
